dpbench/benchmarks/pairwise_distance/pairwise_distance_sycl_native_ext/pairwise_distance_sycl/_pairwise_distance_kernel.hpp

```diff
@@ -0,0 +1,46 @@
+//
+// Copyright 2022 Intel Corp.
+//
+// SPDX - License - Identifier : BSD - 3 - Clause
+///
+/// The files implements a SYCL-based Python native extension for the
+/// pairwise_distance benchmark.
+
+#include <CL/sycl.hpp>
```

Comment:
please use sycl.hpp as CL/sycl.hpp is deprecated and set to be removed in future dpcpp.